rs/nns/governance/canister/canister.rs

```diff
@@ -188,6 +191,32 @@ fn schedule_seeding(duration: Duration) {
     });
 }
 
+thread_local! {
+    // The last neuron whose following was pruned (possibly, trivially, i.e. did
+    // not try to remove anything, because it refreshed recently enough).
+    static PRUNE_FOLLOWING_CHECKPOINT: RefCell<NeuronIdProto> = Default::default();
+}
+
+fn schedule_prune_following(delay: Duration) {
+    if !is_prune_following_enabled() {
+        return;
+    }
+
+    ic_cdk_timers::set_timer(delay, || {
+        let start_checkpoint = PRUNE_FOLLOWING_CHECKPOINT.with(|p| *p.borrow());
+
+        let carry_on = || call_context_instruction_counter() < 10_000_000;
```

Comment:
This instruction count seems very low, as in you may only prune 1 neuron per round once neurons are migrated to stable memory.

It could take a long time to make it through all the neurons at 1 per minute.  It would take 138 days to prune 200k neurons roughly....

Reply:
Done.